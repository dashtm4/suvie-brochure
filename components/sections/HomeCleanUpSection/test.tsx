import { shallow } from 'enzyme'
import toJSON from 'enzyme-to-json'

import { HomeCleanUpSection } from '.'

describe('<HomeCleanUpSection />', () => {
  it('should renders correct <HomeCleanUpSection />', () => {
    const wrapper = shallow(<HomeCleanUpSection />)

    expect(toJSON(wrapper)).toMatchSnapshot()
  })
})
