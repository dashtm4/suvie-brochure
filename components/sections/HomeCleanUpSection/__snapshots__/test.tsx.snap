// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<HomeCleanUpSection /> should renders correct <HomeCleanUpSection /> 1`] = `
<section
  className="relative pt-8 pb-12 text-center md:text-left"
  style={
    Object {
      "backgroundColor": "#f4f3f2",
    }
  }
>
  <Container>
    <Row
      className="items-center justify-between"
    >
      <div
        className="w-full md:px-3 md:w-1/2"
      >
        <VideoCard
          className="mb-8"
          videoSrc="https://embed-ssl.wistia.com/deliveries/30e8629fa1dbc94f7dd3bc479271fb6e2641137e.m3u8"
        />
      </div>
      <Col
        className="w-full md:w-5/12"
      >
        <div
          className="md:max-w-sm"
        >
          <SectionTitle
            alignMd="left"
            className="mb-4 md:mb-2"
          >
            Cleanup takes seconds
          </SectionTitle>
          <div
            className="text-quinary"
          >
            Because Suvie is a water-based cooking appliance, it cleans itself as it cooks. And all the Suvie Pans are dishwasher safe.
          </div>
        </div>
      </Col>
    </Row>
  </Container>
  <Wave
    colorHex="#f4f3f2"
    position="bottom"
    style={
      Object {
        "zIndex": 1,
      }
    }
  />
</section>
`;
